bin/manuscripts

```diff
@@ -60,6 +60,7 @@ def get_params():
     parser.add_argument('-n', '--name', help="Report name")
     parser.add_argument('-m', '--mordred-config', help="Mordred config file")
     parser.add_argument('-p', '--projects', action='store_true', help="Generate per project data")
+    parser.add_argument('-l', '--new-logo', help="Logo file (xyz.pdf)")
```

Comment:
I think just `logo` instead of `new-logo` is shorter, and equally informative. In the help I would say something like "File to use as a logo in the report. Extensions (and formats) allowed: PDF..." (specifying at least some of the extensions allowed, I think at least PDF, EPS, JPEG, PNG and GIF can be used, please check).